package.json

```diff
@@ -27,7 +28,9 @@
     "@types/on-finished": "^2.3.1",
     "chalk": "^2.4.2",
     "on-finished": "^2.3.0",
-    "path-to-regexp": "^3.0.0",
-    "prom-client": "^12.0.0"
+    "path-to-regexp": "^3.0.0"
```

Comment:
What can we do to keep these in alphabetical order automatically..?

Reply:
npm generally does this. Or am I missing something?